build.gradle

```diff
@@ -69,19 +69,19 @@ sourceSets {
 }
 
 sharedLibrary {
-    coreVersion = '2.355' // https://mvnrepository.com/artifact/org.jenkins-ci.main/jenkins-core?repo=jenkins-releases
+    coreVersion = '2.358' // https://repo.jenkins-ci.org/public/org/jenkins-ci/main/jenkins-core/
```

Comment:
It's a little bit hard to locate the latest version with this link. Maybe we keep both the original one and this link in comments?

Reply:
@zelinh ya true, but when you open the link https://repo.jenkins-ci.org/public/org/jenkins-ci/main/jenkins-core/ the contents are not sorted out, so I have added upon the artifcatID and not until version.